pkg/apis/batch/validation/validation.go

```diff
@@ -248,7 +248,7 @@ func validateConcurrencyPolicy(concurrencyPolicy *batch.ConcurrencyPolicy, fldPa
 
 func validateScheduleFormat(schedule string, fldPath *field.Path) field.ErrorList {
 	allErrs := field.ErrorList{}
-	if _, err := cron.ParseStandard(schedule); err != nil {
+	if _, err := cronv3.ParseStandard(schedule); err != nil {
```

Comment:
does this accept any new values, or reject any previously accepted values?

Reply:
No. [this](https://github.com/robfig/cron/commit/470ac64214586d0702fa4de01803228f211f30f2#diff-83eb8e32639d01cf443d6d8bde24c1c8be78766090d8c5f8586c36250cfedca6R146) commit introduces the standard parser which is the same for all versions from 1.1.0 to master